cockatrice/src/dlg_connect.cpp

```diff
@@ -232,6 +234,8 @@ void DlgConnect::previousHostSelected(bool state)
 {
     if (state) {
         saveEdit->setDisabled(true);
+        hostEdit->setDisabled(true);
```

Comment:
disagree with disabling these. If they typo a custom it goes for naught as they have to delete/remake it.

Reply:
I see that problem.
Though, if they mistype the server url they can't change that anyway because we update/compare based on that. Same for name since that field was disabled all the time. They could only fix a wrong port number from within the UI.

Changing the server url does nothing currently, it won't save a new one and doesn't update the existing. Without deleting/editing the ini file the user is basically stuck with what they entered, we never supported editing existing entries from within the UI.

What we could do is to disable url, but enable name to let them edit that.